src/tokenserver/handlers.rs

```diff
@@ -82,7 +90,7 @@ pub async fn get_tokenserver_result(
                 let current_time = start.duration_since(UNIX_EPOCH).unwrap();
                 let expires = current_time + Duration::new(DEFAULT_TOKEN_DURATION, 0);
 
-                expires.as_secs_f64()
+                expires.as_secs()
```

Comment:
On the legacy Tokenserver, the expires timestamp is an integer

Reply:
I was thinking that we were doing something around shifting the time up by 1000 in sync, but that's for the TTL.